### DISCUSSION TITLE
something is wrong with a network card

### DISCUSSION THREAD

**Discussion by codefaztaz** [Security-Onion-Solutions/securityonion#11447] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16 gb

### Storage for /

500j

### Storage for /nsm

100

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have installed securityonion eval version. with ifconfig I have an ip in wan interface but with LAN interface I don't have ip. when I configured the wan y lan I set up in LAN the ip for web access but after instalation I can not access the web portal by lan ip. I can not have internet in my main machine.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Most folks connect their management interface to their LAN and give it a static IP address in the LAN range.

Also see my reply to your other discussion:
https://github.com/Security-Onion-Solutions/securityonion/discussions/11445#discussioncomment-7164174
